src/pages/cli/graphql/authorization-rules.mdx

```diff
@@ -95,7 +95,7 @@ type Todo @model @auth(rules: [{ allow: owner, operations: [create, read, update
 
 Behind the scenes, Amplify will automatically add a `owner: String` field to each record which contains the record owner's identity information upon record creation.
 
-By default, the Cognito user pool's user information is populated into the `owner` field. You can alternatively configure [OpenID Connect as an authorization provider](#using-oidc-authorization-provider).
+By default, the Cognito user pool's user information is populated into the `owner` field. The value that is saved is the `sub` and `username` in the format `<sub>:<username>`. The API will authorize either against the full `<sub>:<username>` value, or `sub` or `username` separately; the API will then return `username`. You can alternatively configure [OpenID Connect as an authorization provider](#using-oidc-authorization-provider).
```

Comment:
@danielleadams - Can we possibly change this line to - **The value saved includes `sub` and `username` in the format `<sub>:<username>`. The API will authorize against the full value of `<sub>:<username>` or `sub` / `username` separately and return `username`.**

Reply:
@arundna - updated